Makefile

```diff
@@ -5,6 +5,7 @@ BACKUP_DIR ?= /tmp/mnesia_backups
 
 images:
 	packer build packer/epoch.json
+	python packer/cleanup-ami-and-snapshots.py
```

Comment:
It looks like this script never returns non-zero exit status. Is this expected?

Reply:
I think it doesn't matter too much as it's a cleanup script - low priority

Reply:
added exit 1 if any exception